src/styles/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$block-size: 300px;
+
+$main-color: #fff;
+
+$colors:
+  "1" #fe0100,
+  "2" #cc0101,
+  "3" #990101,
+  "4" #600,
+  "5" #320001,
+  "6" #010001;
```

Comment:
try to make it more straight - suggest you to use a multiplier with a step of 0.2 with rgb:
```
rgb(0 * 255, 0, 0)
rgb(0.2 * 255, 0, 0)
...
```
etc.